edge/pkg/edged/containers/container_manager.go

```diff
@@ -1122,7 +1122,7 @@ func (cm *containerManager) freeContainer(freeTime time.Time, gcPolicy kubeconta
 		log.LOGGER.Infof("Evaluating Container ID %s for possible garbage collection", record.containerID)
 		if record.lastUsed.Equal(freeTime) || record.lastUsed.After(freeTime) {
 			log.LOGGER.Infof("Container ID %s has lastUsed=%v which is >= freeTime=%v, not eligible for garbage collection", record.containerID, record.lastUsed, freeTime)
-			break
+			continue
```

Comment:
I think continue here make no sense.

Reply:
since we are looping over container records , at each step we evaluate if the container is eligible for GC and if not , then we should continue to process the next container. The break statement would force termination of the loop with just the first container processed which is wrong.

Reply:
Yes, you are right.

Reply:
my pod can not sync status in my test env.  i find this break will force terminate the loop.  but this change maybe can not fix sync problem.。 by the way, i found that `detectTime` param  is useless in  `detectContainers` funtions.  i do not know why paas `detectTime` param to `detectContainers` funtions.

Reply:
I think @rohitsardesai83 is right

Reply:
@rohitsardesai83  you are right, it should be continue here.